quadriga/src/main/java/edu/asu/diging/quadriga/web/QStore.java

```diff
@@ -0,0 +1,78 @@
+package edu.asu.diging.quadriga.web;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.JSONException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import edu.asu.diging.quadriga.exceptions.InvalidDataException;
+import edu.asu.diging.quadriga.exceptions.ParserException;
+import edu.asu.diging.quadriga.message.Message;
+import edu.asu.diging.quadriga.service.IRepositoryManager;
+
+@Controller
+public class QStore {
```

Comment:
this should be in an `api/v0` package and called AddNetworkController or something